pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -19,126 +16,82 @@ import (
 )
 
 func checkIfTrayInstalled() error {
-	/* We want to force installation whenever setup is ran
-	 * as we want the service config to point to the executable
-	 * with which the setup command was issued.
-	 */
-
-	return fmt.Errorf("Ignoring check and forcing installation of System Tray")
+	// Check if tray exists in the startup folder
+	// And version of the tray is matching with expected version
```

Comment:
Is this a TODO? If it is, can you add `TODO`, `FIXME`,. ...before the comment to make it obvious?
Edit: the check is changed in a later commit to stop returning an error unconditionally, but I still think the comment is not 100% correct.
Edit2: this is changed again in a 3rd commit, and the comment then becomes accurate. I'd suggest only adding the comment in the last commit, or removing the comment, the code is easy enough to read (because constants are used, function name is good), so the comment is just repetition?

Reply:
Removed the comment!